pycqed/instrument_drivers/physical_instruments/QuTech_VSM_Module.py

```diff
@@ -0,0 +1,267 @@
+                                   vals=vals.OnOff())
+
+    def add_parameters(self):
+        # VSM properties
+
+        self.add_power_supply_parameters()
+        self.add_parameter('mbbc_state', get_cmd='MBBC?',
+                           vals=vals.Enum('connected', 'disconnected'))
+        self.add_temperature_parameters()
+        self.add_marker_parameters()
+        self.add_qubit_parameters()
+
+        # Command class calibration
+        for pulse in ('GAUSSIAN', 'DERIVATIVE'):
+            for ap in ('ATTENUATION', 'PHASE'):
+                # All channels and modules (no getter)
+                var_name = '_{p}_{a}_raw'.format(p=pulse.lower(), a=ap.lower())
+                var_scpi = ':{p}:{a}:RAW'.format(p=pulse, a=ap)
+                self.add_parameter('calibration' + var_name,
+                                   set_cmd='CALIBRATION' + var_scpi + ' {}',
+                                   vals=vals.Ints())
+                # Per channel, all modules (no getter)
+                for channel in self.channels:
+                    ch_name = '_ch{c}'.format(c=channel)
+                    ch_scpi = ':CHANNEL{c}'.format(c=channel)
+                    self.add_parameter('calibration' + ch_name + var_name,
+                                       set_cmd='CALIBRATION' + ch_scpi +
+                                               var_scpi + ' {}',
+                                       vals=vals.Ints())
+                # Per module, all channels (no getter)
+                for mod in self.modules:
+                    mod_name = '_mod{m}'.format(m=mod)
+                    mod_scpi = ':MODULE{m}'.format(m=mod)
+                    self.add_parameter('calibration' + mod_name + var_name,
+                                       set_cmd='CALIBRATION' + mod_scpi +
+                                               var_scpi + ' {}',
+                                       vals=vals.Ints())
+                # Individual outputs: per (module, channel) pair
+                for channel in self.channels:
+                    for mod in self.modules:
+                        mod_ch_name = '_mod{m}_ch{c}'.format(m=mod, c=channel)
+                        mod_ch_scpi = ':MODULE{m}:CHANNEL{c}'.format(m=mod,
+                                                                     c=channel)
+                        self.add_parameter('calibration' + mod_ch_name +
+                                           var_name,
+                                           get_cmd='CALIBRATION' +
+                                                   mod_ch_scpi +
+                                                   var_scpi + '?',
+                                           set_cmd='CALIBRATION' +
+                                                   mod_ch_scpi +
+                                                   var_scpi + ' {}',
+                                           vals=vals.Ints())
+
+
+if __name__ == '__main__':
```

Comment:
I guess the __name__=='__main__': is there for testing purposes. We generally use drivers as a module and as such this bottom part should be removed to keep the code clean. 
Adding an example to the examples folder with this code snippet would be useful but not required.